main.go

```diff
@@ -49,10 +51,23 @@ func mkconf() Config {
 		}
 	}
 
-	dbstring, exists := os.LookupEnv("GODRINK_DB")
-	if !exists {
-		log.Fatal("No database given, exiting!")
+	dbdriver, driverExists := os.LookupEnv("GODRINK_DBDRIVER")
+	dbstring, connExists := os.LookupEnv("GODRINK_DB")
+	if !driverExists {
+		log.Println("No database driver given, using embedded sqlite.")
+		dbdriver = "sqlite3"
```

Comment:
Constants for the database driver types would be a welcome addition, if `sql.Driver` doesn't expose them.

Reply:
The drivers are registered during runtime, so `sql.Driver` can't expose them. The driver itself has the name defined in a package-private variable, so our own constant it is...

Reply:
makes sense, yeah